pyobis/occurrences/occurrences.py

```diff
@@ -251,8 +251,8 @@ def search(
     :param geometry: [String] Well Known Text (WKT). A WKT shape written
         as either POINT, LINESTRING, LINEARRING
         or POLYGON.
-        Example of a polygon: ((30.1 10.1, 20, 20 40, 40 40, 30.1 10.1)) would
-        be queried as http://bit.ly/1BzNwDq
+        Example of a polygon: ((30.1 10.1, 10 20, 20 40, 40 40, 30.1 10.1)) would
+        be queried as https://api.gbif.org/v1/occurrence/search?geometry=POLYGON%28%2830.1+10.1%2C+10+20%2C+20+40%2C+40+40%2C+30.1+10.1%29%29
```

Comment:
I believe, a valid OBIS URL pointing out to the correct use of `geometry` field would be - `https://api.obis.org/v3/occurrence?geometry=POLYGON%28%2830.1%2010.1%2C%2010%2020%2C%2020%2040%2C%2040%2040%2C%2030.1%2010.1%29%29`.